frontend/components/utils/Pagination.js

```diff
@@ -9,6 +9,9 @@ const Pagination = props => {
   useEffect(() => {
     const ele = document.getElementById(props.elementId);
     const isAtBottom = () => {
+      if (ele === null || ele === undefined) {
```

Comment:
I believe we should just skip entirely the event setting if the element has not been found, right?

Reply:
@froi
> Also, we also should stop trying to load more blocks when backend returns an empty list (try searching for an exact block and see that it keeps trying to load more blocks).

Hmm, but we are dealing with real-time data. So what if the first couple of times, the block wasn't found, but was found later. So we cannot return an empty list then.


> Another one, it would be a better UX if we display "no blocks" if there are no blocks found.

Yes. We should do that. Can I make it in another pr?

Reply:
Ok, that is fine for now. Let’s only address the first comment, and it is good to be merged.

Reply:
Which comment are you talking about?

Reply:
> I believe we should just skip entirely the event setting if the element has not been found, right?

Reply:
@frol Should be fixed now. Can you check again?